app/app.js

```
@@ -184,31 +404,37 @@ Vue.prototype.getAccount = async function () {
+            ))
         }
         break
     case 'trezor':
-        const payload = Vue.prototype.trezorPayload || localStorage.get('trezorPayload')
-        const offset = localStorage.get('offset')
-        account = Vue.prototype.HDWalletCreate(
-            payload,
-            offset
-        )
-        localStorage.set('trezorPayload', { xpub: payload.xpub })
+        const trezorPayload = Vue.prototype.trezorPayload || localStorage.get('trezorPayload')
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)